### PR TITLE
EAGLE-14: Hdfs User command re-assembler

### PR DESCRIPTION
**Pull request by yonzhang**
re-create user command through granular hdfs audit logs

### CODE REVIEW

**Comment by yonzhang** [comment]
EAGLE-14 HDFS User Command Reassembler

**Comment by haoch** [comment]
@yonzhang please make sure format the pull request title with EAGLE-14 to make sure hook with jirap system: https://issues.apache.org/jira/browse/EAGLE-14

File: eagle-security/eagle-security-hdfs-auditlog/src/test/java/org/apache/eagle/security/auditlog/TestSiddhiPattern.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
+
```
Comment (RalphSu) [inline eagle-security/eagle-security-hdfs-auditlog/src/test/java/org/apache/eagle/security/auditlog/TestSiddhiPattern.java]: The license information is missing. 
btw: We might also check in the eclipse format/template into code base.

**Comment by RalphSu** [comment]
Reviewed, see comments above about the license information missing. Looks good to me for other files.

**Comment by yonzhang** [comment]
I have fixed the license header issue, I will merge

**Pull request closed without merging**